src/browser/BrowserOptionDialog.cpp

```diff
@@ -39,6 +39,13 @@ BrowserOptionDialog::BrowserOptionDialog(QWidget* parent)
         "<a href=\"https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/\">Firefox</a>",
         "<a href=\"https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk\">Google Chrome / Chromium / Vivaldi</a>"));
 
+    m_ui->scriptWarningWidget->setVisible(false);
+    m_ui->scriptWarningWidget->setAutoHideTimeout(-1);
+    m_ui->scriptWarningWidget->showMessage(tr("<b>Warning:</b> keepassxc-proxy binary is not found!"
+                                          "<br />Please check your install path for the file, or check your custom path."
+                                          "<br />KeePassXC needs keepassxc-proxy to receive messages from KeePassXC-Browser."
+                                          "<br />Path: "), MessageWidget::Warning);
```

Comment:
Recommended wording:

Warning, the keepassxc-proxy application was not found!
Please check the KeePassXC installation directory or confirm the custom path in advanced options.
Browser integration WILL NOT WORK without the proxy application.
Expected Path: %path%